sql/updates/world/4.3.4/2021_07_23_00_world.sql

```diff
@@ -0,0 +1,360 @@
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Giant Rabid Bear
+SET @ENTRY := 1797;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Rabid Dog
+SET @ENTRY := 1766;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Vile Fin Oracle
+SET @ENTRY := 1908;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Vile Fin Tidehunter
+SET @ENTRY := 1768;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Skitterweb Lurker
+SET @ENTRY := 1781;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Skitterweb Lurker
+SET @ENTRY := 1781;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Bloodfang Scavenger
+SET @ENTRY := 44547;
+UPDATE `creature` SET `MovementType`=1 WHERE `id` = @ENTRY;
+UPDATE `creature` SET `spawndist`=10 WHERE `id` = @ENTRY;
+
+ -- Bat Handler Maggotbreath
+INSERT INTO `creature` VALUES (@CGUID+3, 44825, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1423.08, 1016.46, 52.544, 2.90902, 300, 0, 0, 77490, 0, 0, 0, 0, 0, NULL, 0);
+
+ -- Deathstalker Commander Belmont
+INSERT INTO `creature` VALUES (@CGUID+4, 44789, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1415.07, 1010.24, 52.7315, 2.09687, 300, 0, 0, 2085, 0, 0, 0, 0, 0, NULL, 0);
+
+-- High Apothecary Shana T'veen
+INSERT INTO `creature` VALUES (@CGUID+5, 44784, 0, 0, 0, 1, 0, 1, 169, 0, -1, 0, 1, 1431.55, 1086.89, 60.4768, 3.81387, 300, 0, 0, 276, 590, 0, 0, 0, 0, NULL, 0);
+
+REPLACE INTO `creature_equip_template` VALUES (44365, 1, 2179, 0, 42775, 15595);
+REPLACE INTO `creature_equip_template` VALUES (44640, 1, 47031, 0, 0, 18019);
+REPLACE INTO `creature_equip_template` VALUES (44791, 1, 1481, 0, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44791, 2, 11121, 9661, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44791, 3, 15273, 0, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44791, 4, 36502, 0, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44792, 1, 1481, 0, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44792, 2, 11121, 9661, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44792, 3, 15273, 0, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44792, 4, 36502, 0, 0, 0);
+REPLACE INTO `creature_equip_template` VALUES (44632, 1, 0, 0, 34263, 18019);
+
+REPLACE INTO `creature_template` VALUES (44608, 0, 0, 0, 0, 0, 33996, 0, 0, 0, 'Agatha', '', '', '', 0, 15, 15, 0, 0, 68, 2, 0.6, 1.14286, 1, 1, 0, 2000, 2000, 1, 1, 2, 33536, 2048, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', 0, 2.4, 30, 1, 20, 1, 1, 1, 1, 0, 868, 1, 0, 0, 0, '', 15595);
```

Comment:
Hell no. update fields, do not replace entire entries.

Reply:
Fixed.